packages/esm-service-queues-app/src/queue-table/queue-table.component.tsx

```diff
@@ -119,10 +120,10 @@ export const QueueTableByStatus: React.FC = () => {
 };
 
 interface QueueTableProps {
-  visitedQueueEntriesByLocationAndQueueAndStatus: VisitQueueEntry[];
+  visitQueueEntriesByLocationAndQueueAndStatus: VisitQueueEntry[];
```

Comment:
I would expect a `QueueTable` to take in  a `queueEntries` or `visitQueueEntries` property.  Why are we calling these things `ByLocationAndQueueAndStatus`?  Ideally we'd use the same QueueTable and QueueTable props to render QueueTables in different contexts, with different columns, and with different collections of QueueEntries.  In the context above, you might set the `queueEntries` from `visitQueueEntriesByLocationAndQueueAndStatus`, but that is only this particular use case.

Reply:
I wanted to make it explicit that this `VisitQueueEntry[]` has been filtered. You're right that the filtering can be in different contexts and does not necessarily have to be by Location/Queue/Status. I can renamed it to `queueEntriesToDisplay`.

Reply:
`queueEntriesToDisplay` is fine I guess, but I still prefer just `queueEntries` for readability and consistency.  I don't think anyone should have any expectation that this is every queue entry since the beginning of time that is expected to be passed in as a prop.  If this was always going to have the same usage, it wouldn't be a prop, you would just retrieve them within the component.  Passing it as a prop implies that the queue table takes in a list of queue entries to render, as that's the whole point.

Reply:
If I'm following correctly :) I agree with Mike's logic here.